algebird-util/src/main/scala/com/twitter/algebird/util/UtilAlgebras.scala

```diff
@@ -50,13 +50,25 @@ object UtilAlgebras {
 
   implicit def futureSemigroup[T: Semigroup]: Semigroup[Future[T]] = new ApplicativeSemigroup[T, Future]
   implicit def futureMonoid[T: Monoid]: Monoid[Future[T]] = new ApplicativeMonoid[T, Future]
+
+  implicit def trySemigroup[T: Semigroup]: Semigroup[Try[T]] = new ApplicativeSemigroup[T, Try]
+  implicit def tryMonoid[T: Monoid]: Monoid[Try[T]] = new ApplicativeMonoid[T, Try]
+
+  @deprecated("futureGroup is broken and will be removed at the next minor version bump", since = "0.12.3")
   implicit def futureGroup[T: Group]: Group[Future[T]] = new ApplicativeGroup[T, Future]
+
+  @deprecated("futureRing is broken and will be removed at the next minor version bump", since = "0.12.3")
   implicit def futureRing[T: Ring]: Ring[Future[T]] = new ApplicativeRing[T, Future]
+
+  @deprecated("futureField is broken and will be removed at the next minor version bump", since = "0.12.3")
   implicit def futureField[T: Field]: Field[Future[T]] = new ApplicativeField[T, Future]
 
-  implicit def trySemigroup[T: Semigroup]: Semigroup[Try[T]] = new ApplicativeSemigroup[T, Try]
-  implicit def tryMonoid[T: Monoid]: Monoid[Try[T]] = new ApplicativeMonoid[T, Try]
+  @deprecated("tryGroup is broken and will be removed at the next minor version bump", since = "0.12.3")
```

Comment:
I kept these here to maintain binary compatibility, even though they're broken and untested.